stable/jenkins/values.yaml

```diff
@@ -356,6 +356,17 @@ agent:
   # Allows the Pod to remain active for reuse until the configured number of
   # minutes has passed since the last step was executed on it.
   idleMinutes: 0
+  # Raw yaml template for the Pod. For example this allows usage of toleration for agent pods.
+  # https://github.com/jenkinsci/kubernetes-plugin#using-yaml-to-define-pod-templates
+  # https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  # yamlTemplate: |-
```

Comment:
adding a line above with the empty default value makes it easier to read the values file. Some IDEs also provide autocompletion for values already present in the `values.yaml` of the helm chart.

Reply:
Good to know I'll add an empty value line above this.